src/styles/card.css

```diff
@@ -0,0 +1,105 @@
+
+body {
+  --large-text-color: #060b35;
+  --small-text-color: #616070;
+  --button-color: #00acdc;
+}
+
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
```

Comment:
The task requires you to use SCSS. Rename the file to 'card.scss' and update the styling to use SCSS syntax.